src/main/java/org/swisspush/redisques/util/RedisMonitor.java

```diff
@@ -0,0 +1,107 @@
+        this.vertx = vertx;
+        this.redisProvider = redisProvider;
+        this.period = period * 1000;
+        this.publisher = publisher;
+    }
+
+    public void start() {
+        timer = vertx.setPeriodic(period, timer -> redisProvider.redis().onSuccess(redisAPI -> {
+            redisAPI.info(new ArrayList<>()).onComplete(event -> {
+                if (event.succeeded()) {
+                    collectMetrics(event.result().toBuffer());
+                } else {
+                    log.warn("Cannot collect INFO from redis");
+                }
+            });
+        }).onFailure(throwable -> log.warn("Cannot collect INFO from redis", throwable)));
+    }
+
+    public void stop() {
+        if (timer != 0) {
+            vertx.cancelTimer(timer);
+        }
+    }
+
+    private void collectMetrics(Buffer buffer) {
+        Map<String, String> map = new HashMap<>();
+
+        Splitter.on(System.lineSeparator()).omitEmptyStrings()
+                .trimResults().splitToList(buffer.toString()).stream()
+                .filter(input -> input != null && input.contains(DELIMITER)
+                        && !input.contains("executable")
+                        && !input.contains("config_file")).forEach(entry -> {
+                    List<String> keyValue = Splitter.on(DELIMITER).omitEmptyStrings().trimResults().splitToList(entry);
+                    if (keyValue.size() == 2) {
+                        map.put(keyValue.get(0), keyValue.get(1));
+                    }
+                });
+
+        log.debug("got redis metrics {}", map);
+
+        map.forEach((key, valueStr) -> {
+            long value;
+            try {
+                if (key.startsWith("db")) {
+                    String[] pairs = valueStr.split(",");
+                    for (String pair : pairs) {
+                        String[] tokens = pair.split("=");
+                        if (tokens.length == 2) {
+                            value = Long.parseLong(tokens[1]);
+                            publisher.publishMetric("keyspace." + key + "." + tokens[0], value);
+                        } else {
+                            log.warn("Invalid keyspace property. Will be ignored");
+                        }
+                    }
+                } else if (key.contains("_cpu_")) {
+                    value = (long) (Double.parseDouble(valueStr) * 1000.0);
+                    publisher.publishMetric(key, value);
+                } else if (key.contains("fragmentation_ratio")) {
+                    value = (long) (Double.parseDouble(valueStr));
+                    publisher.publishMetric(key, value);
+                } else {
+                    value = Long.parseLong(valueStr);
+                    publisher.publishMetric(key, value);
+                }
+            } catch (NumberFormatException e) {
+                // ignore this field
+            }
```

Comment:
```patch
   } catch (NumberFormatException e) {
-      // ignore this field
+      log.debug("ignore field '{}' because '{}' doesnt look number-ish enough", key, valueStr);
   }
```